app/components/atomic/galoy-input/galoy-redesigned-input.tsx

```diff
@@ -1,57 +1,119 @@
-import { TextInput } from "react-native"
+import { StyleProp, Text, TextInput, View, ViewStyle } from "react-native"
 
 import { Input, InputProps, makeStyles, useTheme } from "@rneui/themed"
 
 import { ComponentType } from "../../../types/jsx"
 
 const useStyles = makeStyles((theme) => ({
+  ContainerStyle: {
+    paddingBottom: 3,
+    paddingTop: 3,
+    marginLeft: 0,
+    borderRadius: 10,
+    backgroundColor: theme.colors.primary9,
+  },
   inputContainerFocused: {
     borderColor: theme.colors.primary5,
     backgroundColor: theme.colors.white,
-    shadowOffset: { width: 3, height: 2 },
-    shadowRadius: 2,
-    shadowOpacity: 1,
+    marginLeft: -7,
+    marginRight: -7,
+  },
+  errorStateStyle: {
+    borderColor: theme.colors.error5,
+  },
+  labelStyle: {
+    display: "none",
   },
-  captionStyle: {
-    color: theme.colors.grey5,
-    textTransform: "lowercase",
+  errorStyle: {
+    display: "none",
   },
 }))
 
 type GaloyInputProps = {
   initIsFocused?: boolean
   caption?: string
-}
+} & InputProps
 
 const GaloyInputFunctions = (
-  props: InputProps & GaloyInputProps,
+  props: GaloyInputProps,
   ref: React.Ref<TextInput> & React.Ref<React.PropsWithChildren<InputProps>>,
 ) => {
   const { theme } = useTheme()
-  const styles = useStyles(theme)
-  const [isFocused, setIsFocused] = React.useState(props.initIsFocused ?? false)
+  const { containerStyle, ...remainingProps } = props
+  const styles = useStyles(remainingProps)
+  const [isFocused, setIsFocused] = React.useState(remainingProps.initIsFocused ?? false)
+
+  return (
+    <View style={containerStyle}>
+      <LabelComponent
+        props={remainingProps}
+        theme={theme}
+        isFocused={isFocused}
+        labelStyle={remainingProps.labelStyle}
+      />
+      <Input
+        {...remainingProps}
+        labelStyle={styles.labelStyle}
+        errorStyle={styles.errorStyle}
+        containerStyle={isFocused ? styles.ContainerStyle : null}
+        inputContainerStyle={[
+          remainingProps.inputContainerStyle,
+          remainingProps.errorMessage ? styles.errorStateStyle : null,
+          isFocused ? styles.inputContainerFocused : null,
+        ]}
+        placeholderTextColor={theme.colors.grey8}
+        onFocus={(e) => {
+          setIsFocused(true)
+          remainingProps.onFocus?.(e)
+        }}
+        onBlur={(e) => {
+          setIsFocused(false)
+          remainingProps.onBlur?.(e)
+        }}
+        ref={ref}
+      />
+      <CaptionComponent
+        props={remainingProps}
+        isFocused={isFocused}
+        theme={theme}
+        errorStyles={remainingProps.errorStyle}
+      />
+    </View>
+  )
+}
+
+const LabelComponent = ({ props, labelStyle, isFocused, theme }) => {
+  if (!props.label) return null
+  const styles = (): StyleProp<ViewStyle> => {
+    const labelStyles = {
+      marginLeft: isFocused ? 2 : 10,
+      marginBottom: 9,
+      fontWeight: "400",
+      fontSize: 16,
```

Comment:
`fontWeight fontSize lineHeight` can be removed and replaced with the prop `type={'p1'}` on the Text component